README.md

```diff
@@ -23,8 +23,16 @@ $ git flow release publish 0.1.0
 $ git flow release finish 0.1.0
 ```
 
-To kick off the deployment, you'll still need to push the local tags remotely
-`git push --tags`
+After you've completed the `git flow` steps, you'll need to push the changes to your local `master` and `develop` branches back to the main repository. Assuming the name of your default remote for the origin repo is `origin`, you can do that using the following commands:
+
+```bash
+$ git checkout develop
+$ git push origin develop
+$ git checkout master
+$ git push origin master
```

Comment:
I would condense this down to 3 commands (because people are less likely to read paragraphs of text).

```
git push origin develop
git push origin master
# Trigger PyPI deployment
git push --tags
```